### PR TITLE
Add IPV4, V6 and MAC Addresses to Internet

### PR DESCRIPTION
**Pull request by trevershick**
Closes DiUS/java-faker#122

Generate fake IP addresses and MAC addresses.  The location (Internet) of the methods
was taken from the stympy/faker project.  Most of the tests and code were ported from
there as well.

### CODE REVIEW

File: src/test/java/com/github/javafaker/InternetTest.java [hunk 1]
```diff
@@ -105,4 +106,109 @@ public void testPasswordMinLengthMaxLengthIncludeUpperCaseIncludeSpecial() {
...
+                    matchesRegularExpression(one_two_seven),
+                    matchesRegularExpression(one_six_nine),
+                    matchesRegularExpression(one_nine_two),
+                    matchesRegularExpression(one_seven_two)));
+        }
+    }
+
+    @Test
+    public void testPublicIpV4Address() {
+        String ten_dot = "^10\\.";
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/InternetTest.java]: @trevershick could you camelcase the local variables? Thanks

Comment (trevershick) [inline src/test/java/com/github/javafaker/InternetTest.java]: yeah. let me get to that. sorry, i copied the variables from the ruby tests and forgot to update that.

Comment (codingricky) [inline src/test/java/com/github/javafaker/InternetTest.java]: No worries, thanks for your help 😄

Comment (trevershick) [inline src/test/java/com/github/javafaker/InternetTest.java]: done

**Pull request merged**